.github/workflows/build-docker-images.yml

```diff
@@ -11,7 +11,7 @@ on:
         required: true
         type: string
   schedule:
-    - cron: "0 1 * * *"
+    - cron: "17 0 * * *"
```

Comment:
The `17` (which is the scheduled minute in an hour) is to make the actions being triggered as close to the specified time as possible. 

From [doc ](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule):
> The schedule event can be delayed during periods of high loads of GitHub Actions workflow runs. High load times include the start of every hour. If the load is sufficiently high enough, some queued jobs may be dropped. To decrease the chance of delay, schedule your workflow to run at a different time of the hour.